app/mailers/user_mailer.rb

```diff
@@ -92,7 +92,7 @@ def subject_for(key)
       default: [:subject, key.to_s.humanize], app_name: app_name)
   end
 
-  def preview?
-    GovukAdminTemplate.environment_label == "Preview"
+  def integration?
+    Plek.environment == "integration"
```

Comment:
I kinda want to keep Plek fairly dumb and not start wrapping things in `ActiveSupport::StringInquirer`. Open to opinions on this however.